bluetooth/src/main/java/me/aflak/bluetooth/Bluetooth.java

```diff
@@ -47,7 +47,7 @@
     private DiscoveryCallback discoveryCallback;
     private BluetoothCallback bluetoothCallback;
 
-    private ReceiveThread receiveThread;
+    private ArrayList<ReceiveThread> receiveThreads = new ArrayList<>();
```

Comment:
Prefer the use of `List` here : 

List<ReceiveThread> receiveThreads;

and initialize it with the other variables in the `initialize()` method